qutebrowser/app.py

```diff
@@ -423,6 +427,12 @@ def _init_modules(args, crash_handler):
     config.init(qApp)
     save_manager.init_autosave()
 
+    log.init.debug("Initializing keys...")
+    keyconf.init(qApp)
+
+    log.init.debug("Initializing sql...")
+    sql.init(os.path.join(standarddir.data(), 'history.sqlite'))
```

Comment:
Yep, I just tried and with fdba676933e7dd85d1ca61ed6d8f3d534c59f0d5 and 6c534bea6b6127dc8fdb5d13af2a0f6a514d5d8a in, I get:

```python
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/florian/qutebrowser/qutebrowser/__main__.py", line 29, in <module>
    sys.exit(qutebrowser.qutebrowser.main())
  File "/home/florian/qutebrowser/qutebrowser/qutebrowser.py", line 177, in main
    from qutebrowser import app
  File "/home/florian/qutebrowser/qutebrowser/app.py", line 49, in <module>
    from qutebrowser.browser import (urlmarks, adblock, history, browsertab,
  File "/home/florian/qutebrowser/qutebrowser/browser/history.py", line 30, in <module>
    from qutebrowser.misc import objects, sql
  File "/home/florian/qutebrowser/qutebrowser/misc/sql.py", line 25, in <module>
    from PyQt5.QtSql import QSqlDatabase, QSqlQuery
ImportError: No module named 'PyQt5.QtSql'
```